docs/known_issues.rst

```diff
@@ -107,20 +105,22 @@ Either set single array entries or use lists of Quantities::
     >>> a = np.ones(4)
     >>> a[2] = 1*u.cm/u.m
     >>> a
-    array([1., 1., 0.01, 1.])
+    array([1.  , 1.  , 0.01, 1.  ])
 
 ::
 
     >>> a = np.ones(4)
     >>> a[2:3] = [1*u.cm/u.m]
     >>> a
-    array([1., 1., 0.01, 1.])
+    array([1.  , 1.  , 0.01, 1.  ])
 
 Both will throw an exception (albeit not the expected UnitsError), if units do not cancel, e.g.::
```

Comment:
Here we can delete the parenthetical part "(albeit .. )", since a reasonable error is now raised.